### ISSUE TITLE
Math

### ISSUE THREAD

**Issue by soupslurpr** [soupslurpr/BeauTyXT#84]

**Comment by ThatOneCalculator** [comment]
Guessing this issue is related to this?

![Screenshot_20240419-174936.png](https://github.com/soupslurpr/BeauTyXT/assets/44733677/041c248f-4090-40c5-9dfe-b2c6dcf70397)

**Comment by soupslurpr** [comment]
No, this is for the experimental Markdown to Docx feature. To use math with Typst, you need to set the font to the one supporting math. It doesn't do this by default unfortunately, not sure why. You need to do the following for the one included with BeauTyXT or use a custom font file that you have:

#show math.equation: set text(font: "Stix Two Math")

**Comment by lrq3000** [comment]
@soupslurpr please add a message somewhere in the settings about this issue because I'm pretty sure a lot of first-time users may just drop the app believing it doesn't support maths (I almost did).

Otherwise awesome work on this tool, it even supports local bibtex and biblatex files! Pretty cool!

**Comment by soupslurpr** [comment]
@lrq3000 ah yeah that's probably a good idea but I'll first see if it can be fixed by changing some code in the font selection part.

Thanks, and I didn't even know Typst supported that lol!

**Comment by soupslurpr** [comment]
> [!NOTE]
> GPT-6-Astra, responding on behalf of soupslurpr.

[BeauTyXT Reimagined (version 84)](https://github.com/soupslurpr/BeauTyXT/releases/tag/84) is now released. It requires Android 17 / API 37 or newer.

As clarified earlier in this thread, this issue tracked math in the experimental DOCX exporter. That exporter, and the old Typst support discussed here, were not carried forward into Reimagined. Closing this old export task as no longer planned. Reimagined does support a bounded subset of inline and display TeX math in native Markdown preview and formatted printing.
